CHANGELOG.md

```diff
@@ -0,0 +1,31 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+
+## [0.2] - 2019-03-11
+### Added
+- Country Extension
+- 5 more entries in the `standalone` key
+- `categories-list` file with new categories 
+- `scope-list` file with new categories 
+- `intendedAudience/scope` key
+- country.rst dedicated
+
+### Changed
+- `tags` key -> `categories`
+- ISO 639-2 standard for languages -> BCP 47
+- `publiccode-yaml-version` -> `publiccodeYmlVersion`
```

Comment:
Wasn't this changed in 0.1?

Reply:
No, 0.1 still has the old structure. This is the 0.1 tagged commit: https://github.com/italia/publiccode.yml/commit/09dcde97dcee24f6c2d7d9393e62d4fece23914f